touchforms/formplayer/models.py

```diff
@@ -12,6 +12,14 @@
 VERSION_KEY = "version"
 UIVERSION_KEY = "uiVersion"
 
+
+class SqlStatus(models.Model):
+    username = models.CharField(max_length=100, primary_key=True)
```

Comment:
is this arbitrarily set at 100? i don't think we really have a limit so we should either set this to a text field or make the `max_length=255`
<img width="1401" alt="screen shot 2015-08-27 at 1 27 34 pm" src="https://cloud.githubusercontent.com/assets/918514/9527866/913af64a-4cbf-11e5-8fda-2b77f681064d.png">